### ISSUE TITLE
duplicate symbol error

### ISSUE THREAD

**Issue by HansPeterAnders** [Appboy/appboy-ios-sdk#47]
I have made an update with cocoapods from version 2.15.1 to version 2.18.1. If i will use the simulator, the build failed with the following error. 

```
duplicate symbol _zip_copyright in:
    /Users/Swen/Library/Developer/Xcode/DerivedData/dailyme-gwfeetlxyiphhjexlqhwkqsvlwdl/Build/Intermediates/dailyme.build/Debug-iphonesimulator/dailyme.build/Objects-normal/x86_64/zip.o
    /Users/Swen/Documents/iOS/Arbeit/dailyme2/Pods/Appboy-iOS-SDK/AppboyKit/libAppboyKitLibrary.a(ABKZAzip.o)
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by briancaw** [comment]
Hi @HansPeterAnders 

Thanks very much for reporting this.  We have identified the issue and will be strategizing around a fix and release today.  We'll update ASAP when we have a concrete timeline.

Thanks,
Appboy iOS Team

**Comment by briancaw** [comment]
Hey again @HansPeterAnders 

We've just released 2.18.2 which should solve this issue.  Can you please confirm that everything is working properly with this new version?

Thanks,
Appboy iOS Team

**Comment by HansPeterAnders** [comment]
Hey @briancaw 

Everything is working with the new version. Thank you for the quick help.
